### DISCUSSION TITLE
Selectable vs Readonly fields

### DISCUSSION THREAD

**Discussion by magomimmo** [directus/directus#3562] [Archive]
It often happens that you want a field's collection expressing a relationship with other collections to be selectable only, instead of readonly or hidden on details.  What is the best practice to obtain this effect in an interface component like the one used by directus itself. In the image below I can only select a password policy, not creating a new one.

![directus-issue-01](https://user-images.githubusercontent.com/1448905/103481834-876b4f00-4ddd-11eb-89cf-a03743f356b0.png)

**Comment by rijkvanzanten** [comment]
In this case, that's a "regular" dropdown (not relational). There's an open issue to add the missing "hide add/select" feature to the relational interfaces 👍🏻 

https://github.com/directus/directus/issues/3222

**Comment by magomimmo** [comment]
Thanks so much!
